src/scripts/main.js

```diff
@@ -1,3 +1,50 @@
 'use strict';
 
-// write code here
+document.addEventListener('DOMContentLoaded', () => {
+  const table = document.querySelector('table');
+  const headers = table.querySelectorAll('th');
+  const tbody = table.querySelector('tbody');
```

Comment:
Check if the `tbody` element exists. If the table structure is different, this could lead to errors.